test/test_dht.cpp

```diff
@@ -136,6 +136,57 @@ void lazy_from_entry(entry const& e, bdecode_node& l)
 	TEST_CHECK(ret == 0);
 }
 
+void send_dht_request2(node& node, char const* msg, udp::endpoint const& ep
```

Comment:
I agree that the existing one has too many parameters. maybe send_simple_dht_request() or something? I imagine it has some restrictions compared to the other one, in the sense of what kinds of messages it can send.